validator/webui/index.html

```diff
@@ -260,152 +265,199 @@
         },
         checkURL: function(e) {
           this.validURL = !this.$.urlToFetch.invalid;
+        },
+        setURLAndValidate: function(url) {
+          this.$.urlToFetch.value = url;
+          this.$.urlToFetch.validate();
+          this.$.validateButton.click();
         }
       });
     </script>
   </dom-module>
 
+  <!-- Toolbar displayed at the top of the page. This doesn't do much but has a
+       lot of styling, so we isolate it to keep the source for other parts of
+       the page easier to read. -->
+  <dom-module id="styled-toolbar">
```

Comment:
Nit. Why "styled-toolbar" and not just "toolbar" or "ampproject-toolbar"? I'd expect almost everything to be styled so using styled doesn't seem to add much benefit in clarity of what it is.